sirepo/raydata/scan_monitor.py

```diff
@@ -157,6 +158,25 @@ def statuses_for_scans(cls, catalog_name, uids):
         )
 
 
+class _Beamline(abc.ABC):
+    def __init__(self, catalog_name):
+        self.catalog_name = catalog_name
+
+
+class CHX(_Beamline):
+    def notebooks(self):
```

Comment:
This method should live in the base class since it is the same between both classes.